client/src/fluid/FluidAutocomplete.res

```diff
@@ -707,7 +710,11 @@ let selectUp = (a: t): t =>
 
 let isOpened = (ac: t): bool => Option.isSome(ac.index)
 
+let fontSharedStyle = %twc("font-text tracking-wider")
+
 let typeErrorDoc = ({item, validity}: data): Vdom.t<AppTypes.msg> => {
+  let expected = Html.span(list{tw2(fontSharedStyle, %twc("mx-1"))}, list{Html.text("Expected: ")})
```

Comment:
Having the green/orange text have the same foreground as background is a little jarring - can it be changed?

Also, it strikes me that this green/orange isn't from our main color palate - should it be changed to colors from our palate?